Tasks/DockerV0/package.json

```diff
@@ -1,11 +1,15 @@
 {
   "dependencies": {
-    "@types/mocha": "^2.2.5",
-    "@types/node": "^6.0.101",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^10.17.0",
     "@types/q": "^1.5.0",
+    "@types/uuid": "^8.3.0",
     "del": "2.2.0",
-    "azure-pipelines-tasks-docker-common-v2": "1.0.0",
+    "azure-pipelines-tasks-docker-common-v2": "2.0.1-preview.0",
```

Comment:
Add the azure-pipelines-tasks-lib ref too here. Instead of depending on implicitly

Reply:
Added